release_notes/unreleased.md

```diff
@@ -1 +1,2 @@
 **Unreleased**
+* Fixed the bug which made the app impossible to install
```

Comment:
Could you add ticket ID to this changelog if there is one?

Reply:
Done